src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts

```diff
@@ -159,7 +159,11 @@ export class UBSPersonalInformationComponent implements OnInit, OnDestroy {
       )
       .subscribe(() => this.findAllAddresses());
   }
-
+  getStringBetween(str, from, to): string {
+    return str.substring(
+      str.lastIndexOf(from) + from.length,
+      str.lastIndexOf(to));
+  }
```

Comment:
looks not really good 
what the point of this ? 
can we parse string to an array instead?

Reply:
the response from backend is a string with url. this is needed to find url in that string response

Reply:
ok !